app/src/main/java/com/example/tasky/feature_agenda/presentation/event_detail_screen/components/FilterChip.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.tasky.feature_agenda.presentation.event_detail_screen.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.tasky.presentation.theme.BackgroundBlack
+import com.example.tasky.presentation.theme.DarkGray
+import com.example.tasky.presentation.theme.Light2
+import com.example.tasky.presentation.theme.interFont
+
+@Composable
+fun FilterChip(
+    selected: Boolean,
+    text: String,
+    onClick: () -> Unit
+) {
+    Box(
+        modifier = Modifier
+            .size(width = 100.dp, height = 30.dp)
```

Comment:
Could it make more sense to keep the size relative to the width of the screen? On wide screens this might look weird. Instead, you could work with a weight here